network.sh

```diff
@@ -119,7 +119,6 @@ elif [[ "$IP_STACK" = "v6" ]]; then
   export SERVICE_SUBNET_V6=${SERVICE_SUBNET_V6:-"fd02::/112"}
   export NETWORK_TYPE=${NETWORK_TYPE:-"OVNKubernetes"}
   export MIRROR_IMAGES=true
-  export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}/localimages/local-release-image:latest"
```

Comment:
As mentioned above, it seems to me simpler to keep the target repo name the same one, indepedently by the selected backend. The quay one will not support ipv6, so the execution should just fail in that case.

Reply:
See above, regarding the output of the oc-mirror command in the ICSP.